### PR TITLE
Update beansxml FATs

### PR DESCRIPTION
**Pull request by tevans78**
#build

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/apps/invalidBeansXML/InvalidBeansXMLTestServlet.java [hunk 1]
```diff
@@ -8,27 +8,31 @@
...
+import org.junit.Test;
 
-    private static final long serialVersionUID = 1L;
+import componenttest.app.FATServlet;
+
+/**
+ *
+ */
+@WebServlet("/TestServlet")
+public class InvalidBeansXMLTestServlet extends FATServlet {
```
Comment (benjamin-confino) [inline dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/apps/invalidBeansXML/InvalidBeansXMLTestServlet.java:26]: Did github mix two unrelated files or did you rename the class?

**Review by benjamin-confino — APPROVED** [review]

File: dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/apps/invalidBeansXML/InvalidBeansXMLTestServlet.java (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/apps/invalidBeansXML/InvalidBeansXMLTestServlet.java:26]: both!

**Comment by tevans78** [comment]
#run-libby-bot
